benjamingarcia/awesome_repository_GHA
# feat/test GHA merge action 3

- build: test merge check
- build: test merge check test squash and merge
- build: test merge check test squash and merge
